src/main/scala/org/ergoplatform/mining/ErgoMiner.scala

```diff
@@ -458,7 +472,7 @@ class ErgoMiner(ergoSettings: ErgoSettings,
     log.info("Requesting candidate")
     candidateOpt = None
     candidateGenerating = false
-    self ! PrepareCandidate(candidateOpt.map(_.txsToInclude).getOrElse(Seq.empty), reply = false)
+    self ! PrepareCandidate(Seq.empty, reply = false)
```

Comment:
We set `candidateOpt = None` so no need for that construct.

Reply:
True, thanks